src/fetch.ts

```diff
@@ -87,7 +87,7 @@ export function fetchEventSource(input: RequestInfo, {
         let retryTimer = 0;
         function dispose() {
             document.removeEventListener('visibilitychange', onVisibilityChange);

```

Comment:
maybe we need handle `document` too...

Reply:
I have added new commits to handle `document`